dbus-shelly-em-smartmeter.py

```diff
@@ -132,41 +134,44 @@ def _signOfLife(self):
  
   def _update(self):   
     try:
-       #get data from Shelly 1pm
+       #get data from Shelly em
        meter_data = self._getShellyData()
-       
-       config = self._getConfig()
-       str(config['DEFAULT']['Phase'])
-    
-       pvinverter_phase = str(config['DEFAULT']['Phase'])
+      
+      
        
        #send data to DBus
-       for phase in ['L1', 'L2', 'L3']:
-         pre = '/Ac/' + phase
-         
-         if phase == pvinverter_phase:
-           power = meter_data['meters'][0]['power']
-           total = meter_data['meters'][0]['total']
-           voltage = 230
-           current = power / voltage
-           
-           self._dbusservice[pre + '/Voltage'] = voltage
-           self._dbusservice[pre + '/Current'] = current
-           self._dbusservice[pre + '/Power'] = power
-           self._dbusservice[pre + '/Energy/Forward'] = total/1000/60 if power > 0 else 0 
-           
-         else:
-           self._dbusservice[pre + '/Voltage'] = 0
-           self._dbusservice[pre + '/Current'] = 0
-           self._dbusservice[pre + '/Power'] = 0
-           self._dbusservice[pre + '/Energy/Forward'] = 0
-           
-       self._dbusservice['/Ac/Power'] = self._dbusservice['/Ac/' + pvinverter_phase + '/Power']
-       self._dbusservice['/Ac/Energy/Forward'] = self._dbusservice['/Ac/' + pvinverter_phase + '/Energy/Forward']
+       self._dbusservice['/Ac/L1/Voltage'] = meter_data['emeters'][0]['voltage']
+ 
+       current = meter_data['emeters'][0]['power'] / meter_data['emeters'][0]['voltage']
+       self._dbusservice['/Ac/L1/Current'] = current
+       
+       self._dbusservice['/Ac/L1/Power'] = meter_data['emeters'][0]['power']
+       self._dbusservice['/Ac/L1/Energy/Forward'] = (meter_data['emeters'][0]['total']/1000)
+       self._dbusservice['/Ac/L1/Energy/Reverse'] = (meter_data['emeters'][0]['total_returned']/1000)    
        
+        
+       #self._dbusservice['/Ac/Power'] = meter_data['total_power'] # positive: consumption, negative: feed into grid
+       #self._dbusservice['/Ac/L2/Voltage'] = meter_data['emeters'][1]['voltage']
+       #self._dbusservice['/Ac/L2/Current'] = meter_data['emeters'][1]['current']
+       #self._dbusservice['/Ac/L2/Power'] = meter_data['emeters'][1]['power']
+       #self._dbusservice['/Ac/L2/Energy/Forward'] = (meter_data['emeters'][1]['total']/1000)
+       #self._dbusservice['/Ac/L2/Energy/Reverse'] = (meter_data['emeters'][1]['total_returned']/1000) 
+
+       #self._dbusservice['/Ac/L3/Voltage'] = meter_data['emeters'][2]['voltage']
+       #self._dbusservice['/Ac/L3/Current'] = meter_data['emeters'][2]['current']
+       #self._dbusservice['/Ac/L3/Power'] = meter_data['emeters'][2]['power']
+       #self._dbusservice['/Ac/L3/Energy/Forward'] = (meter_data['emeters'][2]['total']/1000)
+       #self._dbusservice['/Ac/L3/Energy/Reverse'] = (meter_data['emeters'][2]['total_returned']/1000) 
```

Comment:
This allows the meter being only on phase 1, which is not intended by me. This should be configurable. Another point is: code should not be commented out in published code, this is bad practice.